vktrace/vktrace_viewer/vktraceviewer_qtracefileloader.cpp

```diff
@@ -217,7 +218,18 @@ bool vktraceviewer_QTraceFileLoader::populate_trace_file_info(vktraceviewer_trac
         pTraceFileInfo->packetCount++;
         fileOffset += packetSize;
 
-        fseek(pTraceFileInfo->pFile, fileOffset, SEEK_SET);
+        seekResult = fseek(pTraceFileInfo->pFile, packetSize - sizeof(uint64_t), SEEK_CUR);
```

Comment:
Are vktraceviewer changes part of the trim feature?
Cant we have trim w/o these changes?